haystack/nodes/audio/whisper_transcriber.py

```diff
@@ -48,12 +49,13 @@ def __init__(
         :param api_key: OpenAI API key. If None, a local installation of Whisper is used.
         :param model_name_or_path: Name of the model to use. If using a local installation of Whisper, set this to one of the following values: "tiny", "small", "medium", "large", "large-v2". If using
         the API, set thsi value to: "whisper-1" (default).
+        :param api_base: The OpenAI API Base url, defaults to `https://api.openai.com/v1`.
```

Comment:
Let's move this to the end of doc strings given that we'll move `api_base` to the end of the arguments.